### ISSUE TITLE
ValueError: Cannot use base_id

### ISSUE THREAD

**Issue by vhmartinezm** [vilarneto/pyrtable#2]
Hey, I'm following your example but I'm getting the following error:

```
    266                     function = functools.partial(function, base_id=base_id)
    267                 else:
--> 268                     raise ValueError('Cannot use base_id')
    269
    270             result['Authorization'] = 'Bearer %s' % function()

ValueError: Cannot use base_id
```

I'm sure the API key and base id are correct because I tried the same using curl

**Comment by vilarneto** [comment]
Hello, could you please try the following signature for `get_api_key`:
```
    @classmethod
    def get_api_key(cls, base_id):
        ...
```
Thanks!

**Comment by vhmartinezm** [comment]
It worked! Thank you!

**Comment by vilarneto** [comment]
I'll fix the example code. Sorry for that!

**Comment by vilarneto** [comment]
Version 0.7.10 should work with or without the `base_id` parameter. Documentation was updated as well.

**Comment by vilarneto** [comment]
Fixed in a1a4f46.
